src/pages/MyMove/Review/Review.test.jsx

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react';
+import { render, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import ConnectedReview from './index';
+
+import { MockProviders } from 'testUtils';
+
+// Mock the summary part of the review page since we're just testing the
+// navigation portion.
+jest.mock('components/Customer/Review/Summary/index', () => 'summary');
+
+describe('Review page', () => {
+  const testProps = {
+    canMoveNext: true,
+    push: jest.fn(),
+  };
+
+  it('renders the Review Page', async () => {
+    const { getByText } = render(
+      <MockProviders>
+        <ConnectedReview {...testProps} />
+      </MockProviders>,
+    );
+    await waitFor(() => {
+      expect(getByText('Review your details')).toBeInstanceOf(HTMLHeadingElement);
+    });
```

Comment:
I think where possible it'd be good to use `getByRole` instead of `getByText` per the recommendations: https://testing-library.com/docs/queries/about#priority

With headings you can even pass a level: https://testing-library.com/docs/queries/byrole/#level

I think you can also use `findBy` instead of `queryBy` which will get you the `await waitFor`. https://testing-library.com/docs/dom-testing-library/api-async#findby-queries

```suggestion
    await findByRole('heading', { level: 1, name: 'Review your details' });
```